plugins/inputs/http_listener_v2/http_listener_v2_test.go

```diff
@@ -59,18 +62,22 @@ func newTestHTTPListenerV2() *HTTPListenerV2 {
 		DataSource:     "body",
 		close:          make(chan struct{}),
 	}
-	return listener
+	return listener, nil
 }
 
-func newTestHTTPAuthListener() *HTTPListenerV2 {
-	listener := newTestHTTPListenerV2()
+func newTestHTTPAuthListener() (*HTTPListenerV2, error) {
+	listener, err := newTestHTTPListenerV2()
+	if err != nil {
+		return nil, err
+	}
 	listener.BasicUsername = basicUsername
 	listener.BasicPassword = basicPassword
-	return listener
+	return listener, nil
 }
 
 func newTestHTTPSListenerV2() *HTTPListenerV2 {
-	parser, _ := parsers.NewInfluxParser()
+	parser := &influx.Parser{}
+	_ = parser.Init()
```

Comment:
I know this does not fail, but I would prefer to convert this function the same way as you converted the other ones above, returning the error...

Reply:
Thanks for pointing this out, I overlooked it.